sympy/core/backend.py

```diff
@@ -0,0 +1,27 @@
+import os
+USE_SYMENGINE = os.getenv('USE_SYMENGINE')
+
+if USE_SYMENGINE:
+    import symengine as sm
+    from symengine import (Symbol, Integer, sympify, sympify as S,
+        SympifyError, exp, log, gamma, sqrt, I, E, pi, Matrix,
+        sin, cos, tan, cot, csc, sec, asin, acos, atan, acot, acsc, asec,
+        sinh, cosh, tanh, coth, asinh, acosh, atanh, acoth,
+        Lambdify as lambdify, symarray, diff, zeros, eye, diag, ones, zeros,
+        expand, UndefFunction as Function, symbols, var)
+    from symengine.lib.symengine_wrapper import (Function as FunctionClass,
+        Add as AddClass, Mul as MulClass, add as Add, mul as Mul,
+        FunctionSymbol as AppliedUndef, Derivative as DerivativeClass,
+        Sin as SinClass, Cos as CosClass, Tan as TanClass)
+else:
+    import sympy as sm
```

Comment:
In `mechanics/models.py` there was a statement like `import sympy as sm`. That's why I initially had this, but these 2 statements in `backend.py` can be removed, because you can just do
`from sympy.core import backend as sm` instead of `import sympy as sm`